climada/test/test_plot.py

```diff
@@ -169,7 +170,79 @@ def test_disc_rates(self):
+                                            'climate_scenario': 'historical',
+                                            'nb_synth_tracks':'10'})
+        haz_future = client.get_hazard('tropical_cyclone',
+                                properties={'country_name': 'Haiti',
+                                            'climate_scenario': 'rcp60',
+                                            'ref_year': str(future_year),
+                                            'nb_synth_tracks':'10'})
+
+        # Create an exposure
+        exp_present = client.get_litpop(country='Haiti')
+        exp_future = copy.deepcopy(exp_present)
+        exp_future.ref_year = future_year
+        n_years = exp_future.ref_year - exp_present.ref_year + 1
+        growth = 1.02 ** n_years
+        exp_future.gdf['value'] = exp_future.gdf['value'] * growth
+        # Create an impact function
+        impf_tc = ImpfTropCyclone.from_emanuel_usa()
+        impf_set = ImpactFuncSet([impf_tc])
+        # Create adaptation measures
+        meas_1 = Measure(
+            haz_type='TC',
+            name='Measure A',
+            color_rgb=np.array([0.8, 0.1, 0.1]),
+            cost=5000000000,
+            hazard_inten_imp=(1, -5),
+            risk_transf_cover=0,
+        )
+
+        meas_2 = Measure(
+            haz_type='TC',
+            name='Measure B',
+            color_rgb=np.array([0.1, 0.1, 0.8]),
+            cost=220000000,
+            paa_impact=(1, -0.10),
+        )
+
+        meas_set = MeasureSet(measure_list=[meas_1, meas_2])
+        # Create discount rates
+        year_range = np.arange(exp_present.ref_year, exp_future.ref_year + 1)
+        annual_discount_zero = np.zeros(n_years)
+        discount_zero = DiscRates(year_range, annual_discount_zero)
+        # Wrap the entity together
+        entity_present = Entity(exposures=exp_present, disc_rates=discount_zero,
+                                impact_func_set=impf_set, measure_set=meas_set)
+        entity_future = Entity(exposures=exp_future, disc_rates=discount_zero,
+                                impact_func_set=impf_set, measure_set=meas_set)
+        # Create a cost benefit object
+        costben = CostBenefit()
+        costben.calc(haz_present, entity_present, haz_future=haz_future,
+                    ent_future=entity_future, future_year=future_year,
+                    imp_time_depen=1, save_imp=True)
+
+        # Call the plotting functions
+        costben.plot_cost_benefit()
+        costben.plot_event_view((25, 50, 100, 250))
+        costben.plot_waterfall_accumulated(haz_present, entity_present, entity_future)
+        ax = costben.plot_waterfall(haz_present, entity_present,
+                                    haz_future, entity_future)
+        costben.plot_arrow_averted(axis = ax, in_meas_names=['Measure A', 'Measure B'],
+                                    accumulate=True)
+        CostBenefit._plot_list_cost_ben(cb_list = [costben])
```

Comment:
`_plot_list_cost_ben` is a private method that is already called by `plot_cost_benefit`. I should not be necessary to call it explicitly to increase the coverage.
```suggestion
```